packages/matchbox/src/components/Box/Box.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import { border, color, flexbox, grid, layout, position, shadow, space, typography, compose } from 'styled-system';
+import propTypes from '@styled-system/prop-types';
+import PropTypes from 'prop-types';
+
+const system = compose(
+  border,
+  color,
+  flexbox,
+  grid,
+  layout,
+  position,
+  shadow,
+  space,
+  typography
+);
```

Comment:
Would there be a separate PR for incorporating these in to existing components/maybe refactoring styling with styled components?

Reply:
I'm not 100% sure how much of an effort moving everything to styled-components will be. But for now my plan is to use primitives (`Box` and maybe another one called `Text`) in other components, and see how far that gets us. Most likely we'll be doing a separate PR for each component starting with smaller ones to keep review light